src/components/TabTraining.vue

```diff
@@ -9,6 +9,13 @@
     <h2 class="training">Distributed Training (NCCL backend)</h2>
```

Comment:
```suggestion
    <h2 class="training">Distributed Training</h2>
```
Let's remove that nccl is not only supported backend